.gitignore

```diff
@@ -40,3 +40,4 @@ linkchecker-out.html
 .local
 .yarnrc
 yarn.lock
+package.json
```

Comment:
looks odd to ignore this file ?

Reply:
We don't need it. It's presence is misleading because we don't use it or maintain it, but it could lead people to think they should install Antora locally rather than using the container.